src/graphql/tests/joke.test.ts

```diff
@@ -0,0 +1,31 @@
+import { graphql } from 'graphql';
+import schema from '../schema';
+
+const query = `
+  query Q {
+    joke {
+      text
+    }
+  }
+`;
+const rootValue = {};
+const context = {};
+
+describe('joke schema', () => {
+  it('should answer with a joke text containing \'Chuck Norris\' words', async () => {
```

Comment:
this scenario is redundant if we do snapshot match already, leave the content assertions to the snapshot engine